R/stat-bin.R

```diff
@@ -31,7 +31,7 @@
 #'   density  = "density of points in bin, scaled to integrate to 1.",
 #'   ncount   = "count, scaled to a maximum of 1.",
 #'   ndensity = "density, scaled to a maximum of 1.",
-#'   width    = "widths of bins."
+#'   width    = "widths of bins. Use with `after_stat(count / width)` to obtain bars with _areas_ representing counts (e.g., with non-equal-width bins). See example."
```

Comment:
Could we move this addition to the 'details' or 'notes' section of the documentation? It feels a bit out of place as part of this list.

Reply:
Maybe we can add to `@details` instead: 

> By default, the _height_ of the bars represent the counts within each bin. However, the are situations where this behavior might produce misleading plots (e.g., when non-equal-width bins are used), in which case it might be preferable to have the _area_ of the bars represent the counts (by setting `aes(y = after_stat(count / width))`). See example below.

What do you think?

Reply:
I like it! Just mind the typo in 'However, *there* are ...'